.gitignore

```diff
@@ -1,3 +1,4 @@
 /oci-create-runtime-bundle
 /oci-unpack
 /oci-image-validate
+/oci-generate
```

Comment:
The current position of `oci-unpack` is unfortunate, but I think we should alphabetize these entries (and the entries in the Makefile's `TOOLS`).